criu/uffd.c

```diff
@@ -262,14 +262,18 @@ static int uffd_api_ioctl(void *arg, int fd, pid_t pid)
 	return ioctl(fd, UFFDIO_API, uffdio_api);
 }
 
-int uffd_open(int flags, unsigned long *features)
+int uffd_open(int flags, unsigned long *features, bool userfaultfd_ok_to_fail)
 {
 	struct uffdio_api uffdio_api = { 0 };
 	int uffd;
 
 	uffd = syscall(SYS_userfaultfd, flags);
 	if (uffd == -1) {
-		pr_perror("Lazy pages are not available");
+		if (!userfaultfd_ok_to_fail) {
```

Comment:
Is there a good reason why you do `!userfaultfd_ok_to_fail` instead of `userfaultfd_ok_to_fail`? If not, change it and reverse the order of if/else.

Reply:
I see how my use of `!userfaultfd_ok_to_fail` is poor coding practice. I'll make this change now.